config/etfuturum/enchantspotions.cfg

```diff
@@ -0,0 +1,43 @@
+# Configuration file
+
+##########################################################################################################
+# enchantments
+#--------------------------------------------------------------------------------------------------------#
+# Settings for enchantments.
+# By default, only IDs up to 255 are allowed, only assign a larger value if an ID extension is present.
+##########################################################################################################
+
+enchantments {
```

Comment:
ah you probably also need to check potion and enchantment IDs. NEI dump should give those too.

Reply:
Done. Updated IDs